modules/SofaGeneralEngine/MeshBarycentricMapperEngine.h

```diff
@@ -83,7 +83,7 @@ class MeshBarycentricMapperEngine : public core::DataEngine
         {
             msg_warning() << "input data 'InputMeshName' changed for 'topology', please update your scene (see PR#1487)";
         }
-        MeshBarycentricMapperEngine::parse(arg);
+        core::DataEngine::parse(arg);
```

Comment:
does it make a diff to do that call or ```Inherit1::parse(arg);``` ?

Reply:
Inherit1 ?
You mean to make a typedef Inherit1 = core::DataEngine and calling it ?

Reply:
my bad I thought it was a smarter implementation. 
ok for me